Packs/PANWComprehensiveInvestigation/Playbooks/Palo_Alto_Networks_-_Hunting_And_Threat_Detection_README.md

```diff
@@ -34,10 +43,11 @@ This playbook does not use any integrations.
 | IPAddresses | List of IP addresses. |  | Optional |
 | URLDomain | List of domains or urls. |  | Optional |
 | InternalRange | A list of internal IP ranges to check IP addresses against. The list should be provided in CIDR notation, separated by commas. An example of a list of ranges would be: "172.16.0.0/12,10.0.0.0/8,192.168.0.0/16" \(without quotes\). If a list is not provided, will use default list provided in the IsIPInRanges script \(the known IPv4 private address ranges\). |  | Optional |
-| InternalDomainName | The organizations internal domain name. This is provided for the script IsInternalHostName that checks if the detected host names are internal or external if the hosts contain the internal domains suffix. For example demisto.com. If there is more than one domain, use the \| character to separate values such as \(demisto.com\|test.com\) |  | Optional |
+| InternalDomainName | The organization's internal domain name. This is provided for the script IsInternalHostName that checks if the detected host names are internal or external if the hosts contain the internal domains suffix. For example demisto.com. If there is more than one domain, use the \| character to separate values such as \(demisto.com\|test.com\) |  | Optional |
 | InternalHostRegex | This is provided for the script IsInternalHostName that checks if the detected host names are internal or external. If the hosts match the organization's naming convention. For example the host testpc1 will have the following regex \\w\{6\}\\d\{1\} |  | Optional |
```

Comment:
@karinafishman this is where the partial sentence is 'If the hosts match the organization's naming convention' - it's not something new, was already in the file